railgun/pkg/config/config.go

```diff
@@ -136,7 +135,7 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.StringVar(&c.LeaderElectionID, "election-id", "5b374a9e.konghq.com", `Election id to use for status update.`)
 	flagSet.StringSliceVar(&c.FilterTags, "kong-admin-filter-tag", []string{"managed-by-ingress-controller"}, "The tag used to manage and filter entities in Kong. This flag can be specified multiple times to specify multiple tags. This setting will be silently ignored if the Kong instance has no tags support.")
 	flagSet.IntVar(&c.Concurrency, "kong-admin-concurrency", 10, "Max number of concurrent requests sent to Kong's Admin API.")
-	flagSet.StringVar(&c.WatchNamespace, "watch-namespace", corev1.NamespaceAll,
+	flagSet.StringSliceVar(&c.WatchNamespaces, "watch-namespace", nil,
```

Comment:
nit: watch-namespaces

Reply:
The name of the flag must remain `watch-namespace` for compatibility with KIC 1.x.
In 1.x, watch-namespace accepted only a single value.

I've opted for renaming the config variable name though, to underline the fact that now there can be any number of watched namespaces.